test/molecule-role/molecule/integrations/tests/test_docker.py

```diff
@@ -0,0 +1,30 @@
+import json
+import os
+
+from testinfra.utils.ansible_runner import AnsibleRunner
+
+import util
+
+testinfra_hosts = AnsibleRunner(os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('agent-integrations-mysql')
+
+
+def test_container_restart_events(host):
+    hostname = host.ansible.get_variables()["inventory_hostname"]
+    url = "http://localhost:7070/api/topic/sts_generic_events?limit=1000"
+
+    def wait_for_events():
+        data = host.check_output("curl \"%s\"" % url)
+        json_data = json.loads(data)
+
+        for message in json_data["messages"]:
```

Comment:
Use `test_event_data` from the integrations test. Move that function to `util` so that it can be used across all tests